snuba/query/functions.py

```diff
@@ -144,6 +144,7 @@ def is_aggregation_function(func_name: str) -> bool:
     "toUInt64",
     "toFloat64",
     # dates and times
+    "toStartOfMinute",
```

Comment:
This isn't used anywhere yet that I can tell in our Sentry queries, but it's used in tests, so I think it's fair to add